src/components/App/App.tsx

```diff
@@ -1,30 +1,37 @@
-import { useState } from 'react';
-import Card from '../Card';
-import './App.css';
+import { useState } from "react";
+import Card from "../Card";
+import "./App.css";
 
 function App() {
   const [isDarkModeActive, setIsDarkModeActive] = useState<boolean>(false);
   const switchModes = (mode: any) => {
```

Comment:
Mode should be typed, check enum typing with typescript, then it will be possible to also replace mode constants with variables